src/coprocessor/dag/expr/builtin_string.rs

```diff
@@ -3259,4 +3283,43 @@ mod tests {
             assert_eq!(got, exp);
         }
     }
+
+    #[test]
+    fn test_instr() {
+        let cases: Vec<(&str, &str, i64)> = vec![
+            ("a", "abcdefg", 1),
+            ("0", "abcdefg", 0),
+            ("c", "abcdefg", 3),
+            ("F", "abcdefg", 6),
+            ("cd", "abcdefg", 3),
+            (" ", "abcdefg", 0),
+            ("", "", 0),
```

Comment:
It seems not correct in mysql 5.6.25

```
mysql> SELECT INSTR('', '');
+---------------+
| INSTR('', '') |
+---------------+
|             1 |
+---------------+
1 row in set (0.00 sec)
```

Reply:
Already fixed. Thanks for the review.

Reply:
Does https://github.com/tikv/tikv/pull/4340 have the same problem?

Reply:
@zhangjinpeng1987 I'm sorry. They have the same problem. I didn't test it in mysql. Unfortunately, I delete the origin branch. Should I create a new pull request for it?

Reply:
Yes, you are welcome.